src/app/components/data-entry-grid/index.html

```diff
@@ -188,6 +197,32 @@ <h4>Read-only mode</h4>
             Specifies that cells the column will display the currently selected value's name property by default. If the autocomplete needs to show a different property or needs to be formatted in any way, you can <a href="https://www.ag-grid.com/javascript-grid-value-formatters/">define a <sky-code>valueFormatter</sky-code></a> on the column definition.
           </p>
         </sky-demo-page-property>
+
+        <sky-demo-page-property
+          propertyName="SkyCellType.Currency"
+          isOptional="true"
+        >
+          <h4>Edit mode</h4>
+          <p>
+            Specifies that cells in the column will be edited as a dollar amount.
```

Comment:
```suggestion
            Specifies that cells in the column will be edited as a currency amount.
```
Not sure if it will always be dollars... is it location aware?